src/vos/vos_common.c

```diff
@@ -804,11 +804,11 @@ vos_self_fini_locked(void)
 		self_mode.self_xs_ctxt = NULL;
 	}
 
-	vos_self_nvme_fini();
 	if (!bio_nvme_configured(SMD_DEV_TYPE_META))
 		vos_db_fini();
 	else
 		lmm_db_fini();
+	vos_self_nvme_fini();
```

Comment:
bio_nvme_fini() is missed in engine mode, could you add it back in this PR? Thanks.

Reply:
sure